src/effect_main.c

```diff
@@ -1193,7 +1193,7 @@ u8 sub_800EC74(void)
 
 bool8 sub_800EC84(s32 moveId)
```

Comment:
Looks like this function name can be labeled as well now that we know what the field is.